nautobot/dcim/tables/devices.py

```diff
@@ -248,6 +304,13 @@ class Meta(BaseTable.Meta):
         order_by = ("device", "name")
 
 
+class ModularDeviceComponentTable(DeviceComponentTable):
+    module = tables.Column(linkify=True)
+
+    class Meta(DeviceComponentTable.Meta):
+        order_by = ("device", "module", "name")
```

Comment:
Curious why this is needed - doesn't the `order_by` on the model handle this already?

Reply:
This might actually be a bug. It appears that when you let the table do the sorting it only accounts for the values of the fields, ie: order by `device.pk`, `module.pk`, `name.pk` instead of the ordering of the database fields.

Reply:
The number of database queries is the same too. I'm removing these